deployment/ansible/roles/raster-foundry.jenkins/tasks/main.yml

```diff
@@ -24,3 +29,14 @@
       -v /etc:/etc
       spotify/docker-gc
     state: "present"
+
+- name: Restart Jenkins every 3 hours
+  cron:
+    name: "jenkins-restart"
+    user: "{{ jenkins_name }}"
+    hour: "*/3"
+    minute: "0"
+    job: >
+      /usr/bin/java -jar {{ jenkins_jar_path }}
```

Comment:
Jenkins was just in a failed state and invoking this command manually helped reset it.